setup/web.sh

```diff
@@ -122,6 +122,17 @@ cat conf/mozilla-autoconfig.xml \
 	 > /var/lib/mailinabox/mozilla-autoconfig.xml
 chmod a+r /var/lib/mailinabox/mozilla-autoconfig.xml
 
+# Create a generic mta-sts.txt file which is exposed via the
+# nginx configuration at /.well-known/mta-sts.txt
+# more documentation is available on: 
+# https://www.uriports.com/blog/mta-sts-explained/
+
+cat conf/mta-sts.txt \
+        | sed "s/MODE/$MTA_STS/" \
+        | sed "s/PRIMARY_HOSTNAME/$PRIMARY_HOSTNAME/" \
```

Comment:
The spec says:

> In the case of Internationalized Domain Names [RFC5891], the "mx" value MUST specify the Punycode-encoded A-label [RFC3492] to match against, and not the Unicode-encoded U-label.

I am not sure if PRIMARY_HOSTNAME could be the Unicode form here or not. Need to check.

Reply:
With this one, i need some help; I'm completely unfamiliar with punycode/unicode.

Reply:
The puny code is added with idn2 package (which also installs). Because it's only used in a bash script, i used a native package instead of writing something in python.

Thanks to some help from "Mustard"